style.css

```diff
@@ -382,3 +365,202 @@ form textarea {
+  .toggleNav {
+    display: none;
+  }
+
+  nav ul {
+    display: flex;
+    list-style-type: none;
+  }
+
+  nav ul li {
+    margin: 10px;
+    padding: 10px;
+    font-family: Poppins, sans-serif;
+    margin-right: 70px;
+    font-size: 120%;
+  }
+
+  nav ul li a {
+    color: #344563;
+  }
+
+  .br {
+    display: none;
+  }
+
+  .container {
+    margin-left: 13vw;
+    margin-right: 32vw;
+  }
+
+  .h1 {
+    line-height: 60px;
+    letter-spacing: 0.38px;
+  }
+
+  .icons {
+    display: grid;
+    grid-template-columns: repeat(5, 20%);
+    list-style: none;
+    text-decoration: none;
+    padding-top: 20px;
+    align-items: flex-start;
+    width: 180px;
+  }
+
+  .connect {
+    font-size: 20px;
+  }
+
+  .card-w1 {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    height: 496px;
+    width: 1156px;
+    margin-top: 142px;
+    margin-bottom: 140px;
+    margin-left: 8vw;
+  }
+
+  .image-section img {
+    width: 42vw;
+    margin-top: 1px;
+  }
+
+  .card-w2 {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: center;
+    align-items: center;
+    height: 496px;
+    width: 1156px;
+    margin-top: 142px;
+    margin-bottom: 140px;
+    margin-left: 8vw;
+  }
+
+  .about-myself {
+    display: grid;
+    background: url(./img/about.png) no-repeat top/cover;
+    grid-template-columns: 50% 50%;
+    border-top: 1px solid white;
+    list-style: none;
+  }
+
+  .card-img {
+    margin-bottom: 55px;
+    margin-left: 25px;
+  }
+
+  .card-text {
+    height: 400px;
+    margin-left: 20px;
+  }
+
+  .titlep1 h2 {
+    margin-bottom: 15px;
+  }
+
+  .description {
+    margin-bottom: 25px;
+  }
+
+  .icons2 {
+    margin-bottom: 20px;
+  }
+
+  .img7 {
+    margin-right: 15px;
+  }
+
+  .card-about {
+    margin-left: 7vw;
+    margin-right: 5vw;
+    margin-bottom: 50px;
+  }
+
+  .title-about {
+    padding-top: 15px;
+  }
+
+  .about-text {
+    margin-top: 55px;
+  }
+
+  .open-menu {
+    display: flex;
+    flex-direction: column;
+    height: 100px;
+    width: 100px;
+  }
+
+  .horizontal {
+    display: flex;
+    flex-direction: row;
+    margin-left: 9vw;
+  }
+
+  .background-div {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .form-container {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .containerxx {
+    display: flex;
+    flex-direction: column;
+    height: 680px;
+    justify-content: center;
+    padding-bottom: 90px;
+  }
+
+  form {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-bottom: 50px;
+  }
+
+  form ul {
+    text-decoration: none;
+    list-style: none;
+  }
+
+  form textarea {
+    margin-left: 9px;
+  }
+
+  .contact-par {
+    margin-left: 33vw;
+    margin-right: 33vw;
+    line-height: 200%;
+    font-size: 20px;
+  }
+
+  .contact-title {
+    margin-top: 100px;
+  }
+}
+
+@media screen and (max-width: 922px) {
```

Comment:
- [ ] Also you don't need a media query for display: none the nav, you can add that to the normal mobile version and then display it on the media query for the desktop